src/main/java/net/snowflake/client/core/UUIDUtils.java

```diff
@@ -0,0 +1,31 @@
+package net.snowflake.client.core;
+
+import java.util.UUID;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class UUIDUtils {
+    static Boolean newUUIDEnabled = true;
+
+    public static UUID getUUID() {
```

Comment:
Suggestion to rename this method for consistency:
```suggestion
    public static UUID randomUUID() {
```